Sources/TensorFlow/Loss.swift

```diff
@@ -53,6 +53,19 @@ public func meanAbsoluteError<Scalar: TensorFlowFloatingPoint>(
     return abs(expected - predicted).mean()
 }
 
+/// Computes the mean absolute percentage error between predictions and expectations.
+///
+/// - Parameters:
+///   - predicted: Predicted outputs from a neural network.
+///   - expected: Expected values, i.e. targets, that correspond to the correct output.
+@differentiable(wrt: predicted)
+public func meanAbsolutePercentageError<Scalar: TensorFlowFloatingPoint>(
+    predicted: Tensor<Scalar>, expected: Tensor<Scalar>
+) -> Tensor<Scalar> {
+    let diff = abs((expected - predicted) / abs(expected))
```

Comment:
Just wondering. Could we avoid the call to abs() in the denominator?

Reply:
@jon-tow I'm afraid not if you don't take the absolute value and you have a negative sign in one of your values then your percentages might no longer be uniform.

Reply:
If you remove the abs() in the denominator of line 65, then the code is computationally equivalent to the equation you've commented above.

The code currently stands as: 
     abs(num / abs(denom))
The abs(denom) seems redundant. How does this break some uniformity?

Reply:
A variety of loss functions have to be clipped. I'm waiting for #198 to pass to update the loss functions, in that case an abs function is needed. I guess we might not need the abs in this PR. I'll remove it in a while.

Reply:
Ah I see. Thanks for the quick response and explanation. 👍

Reply:
Thanks for pointing it out.